pkg/cluster/spec/validate.go

```diff
@@ -307,6 +307,64 @@ Please change to use another port or another host.
 	return nil
 }
 
+// TiKVLabelError indicates that some TiKV servers don't have correct labels
+type TiKVLabelError struct {
+	TiKVInstances map[string][]error
```

Comment:
Prefer to use `slice` to keep the result stable.

Reply:
To make a stable output, I think we should sort the error in the `Error` [method](https://github.com/pingcap/tiup/pull/800/files#diff-3ee04046ce3e73cd5ee6020d945d17f4R316) instead of using a slice here. Because the [origin labels data](https://github.com/pingcap/tiup/pull/800/files#diff-ae7f1374cdcf99ec34f0f79ba4d51b08R127) we fetched is a map, so even if we make it a slice, it's still unstable